homeassistant/components/decora_wifi/light.py

```diff
@@ -171,3 +135,55 @@ def update(self) -> None:
             self._switch.refresh()
         except ValueError:
             _LOGGER.error("Failed to update myLeviton switch data")
+
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return the device info."""
+        return DeviceInfo(
+            identifiers={
+                # Serial numbers are unique identifiers within a specific domain
+                (DOMAIN, self.unique_id)
+            },
+            name=self.name,
+            manufacturer=self._switch.manufacturer,
+            model=self._switch.model,
+            sw_version=self._switch.version,
+            serial_number=self._switch.serial,
+        )
+
+
+class DecoraWifiAsyncClient:
```

Comment:
Can we move this to or `__init__.py` or a separate file

Reply:
ty fixed in https://github.com/home-assistant/core/pull/104573/commits/0566f258bb2bd99a7231282c6f55c0e937d508dc